openpype/hosts/flame/api/lib.py

```diff
@@ -1,12 +1,27 @@
 import sys
 import os
+import re
+import json
 import pickle
 import contextlib
 from pprint import pformat
-
+from .constants import (
+    MARKER_COLOR,
+    MARKER_DURATION,
+    MARKER_NAME,
+    COLOR_MAP,
+    MARKER_PUBLISH_DEFAULT
+)
 from openpype.api import Logger
 
-log = Logger().get_logger(__name__)
+log = Logger.get_logger(__name__)
+
+
+class CTX:
+    # singleton used for passing data between api modules
+    app_framework = None
+    apps = []
```

Comment:
I have to say that I don't know what `apps` stands for. If it's some kind of app in flame could it be renamed to `flame_apps` or something more explaining what is stored there?